java/hello-world/README.md

```diff
@@ -3,20 +3,132 @@
 This is a simple application that demonstrates using the native HBase API
 to connect to and interact with Cloud Bigtable.
 
```

Comment:
Added. I used a tool called doctoc. I'm considering making a bot just for README updates like this. There are plenty of areas where we could use templates for repeated steps.